extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -181,15 +191,36 @@ export class AmpGeo extends AMP.BaseElement {
         user().assert(
             isArray(ISOCountryGroups[group]),
             `${errorPrefix}[${group}] must be an array`);
-        ISOCountryGroups[group] = ISOCountryGroups[group]
-            .map(country => country.toLowerCase());
-        if (ISOCountryGroups[group].includes(this.country_)) {
+
+        if (this.checkGroup_(ISOCountryGroups[group])) {
           this.matchedGroups_.push(group);
         }
       });
     }
   }
 
+  /**
+   * checkGroup_() does this.country_  match the group
+   * @param {!Array<string>} countryGroup The group to match against
+   * @return {boolean}
+   */
+  checkGroup_(countryGroup) {
+    let presetCountries = [];
+    countryGroup = countryGroup.map((country, idx) => {
+      if (/^preset-/.test(country)) {
+        user().assert(
+            isArray(ampGeoPresets[country]),
+            `<${TAG}> preset ${country} not found`);
+        delete countryGroup[idx];
+        presetCountries = presetCountries.concat(ampGeoPresets[country]);
+        return;
```

Comment:
This inserts an `undefined` into the output.

Reply:
Fixed in #16451